src/cluster_copier.cc

```diff
@@ -336,23 +337,49 @@ namespace grpc_labview {
 
     //---------------------------------------------------------------------
     //---------------------------------------------------------------------
+    int GetLVEnumValueFromProtoValue(int protoValue, std::shared_ptr<EnumMetadata> enumMetadata)
+    {
+        int value = 0;
+        auto lvValue = enumMetadata->ProtoEnumToLVEnum.find(protoValue);
+        if (lvValue != enumMetadata->ProtoEnumToLVEnum.end())
+            value = (lvValue->second).front(); // Since one proto value can be mapped to multiple LV enum values, so always return the first element.
+        else
+        {
+            // Throw error
```

Comment:
throw exception for calling code , something like invalid enum value

Reply:
Fixed.